src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -258,7 +258,7 @@ public virtual ObjectResult StatusCode(int statusCode, object value)
         }
 
         /// <summary>
-        /// Creates a <see cref="ContentResult"/> object by specifying a <paramref name="content"/> string.
+        /// Creates a <see cref="ContentResult"/> object (HTTP 200) by specifying a <paramref name="content"/> string.
```

Comment:
I want to homoginize these with the existing status code messages, but it's kind of hard to fit OK (200) in. Wording suggestions? My best idea was `...specifying a <paramref name="content"/> string with OK (200).` but even that seemed kind of stilted.